website/src/components/Chat/ChatSection.tsx

```diff
@@ -1,21 +1,35 @@
+import { useEffect } from "react";
 import { FormProvider, useForm } from "react-hook-form";
 import { ChatConfigForm } from "src/types/Chat";
 
 import { ChatConfigSummary } from "./ChatConfigSummary";
 import { useChatContext } from "./ChatContext";
 import { ChatConversation } from "./ChatConversation";
 
+const CHAT_CONFIG_KEY = "CHAT_CONFIG";
+
 export const ChatSection = ({ chatId }: { chatId: string }) => {
   const { modelInfos } = useChatContext();
 
+  const oldConfig = localStorage.getItem(CHAT_CONFIG_KEY);
```

Comment:
Isn't this crash on the server side?

Reply:
It is weird why it did not crash on my side or in the CI pipeline?

I added a check for it